go.mod

```diff
@@ -538,7 +538,7 @@ replace (
 	golang.org/x/lint => golang.org/x/lint v0.0.0-20190930215403-16217165b5de
 	golang.org/x/mobile => golang.org/x/mobile v0.0.0-20201217150744-e6ae53a27f4f
 	golang.org/x/mod => golang.org/x/mod v0.8.0
-	golang.org/x/net => golang.org/x/net v0.8.0
+	golang.org/x/net => golang.org/x/net v0.17.0
```

Comment:
Oh, actually, you need to pin each golang dep individually on 1.25. We made that easier in 1.26+

Reply:
Ah yes, done @liggitt